src/build/mod.rs

```diff
@@ -5,4 +5,4 @@ pub mod sphere;
 pub mod tenscript;
 pub mod example;
 pub mod brick;
-pub mod brick_proto;
+pub mod brick_proto;
```

Comment:
Is there still a reason for this mod to exist? I would just fold it into `brick`